ext/civicrm_admin_ui/ang/afsearchAssignUsersToRoles.aff.html

```diff
@@ -0,0 +1,9 @@
+<div af-fieldset="">
+  <div class="af-markup">
+    <div class="help">
+    <p>ACLs allow you to control access to CiviCRM data. An ACL consists of an <strong>Operation</strong> (e.g. &#39;View&#39; or &#39;Edit&#39;), a <strong>set of data</strong> that the operation can be performed on (e.g. a group of contacts), and a <strong>Role</strong> that has permission to do this operation. Refer to the <a class="crm-doc-link no-popup" href="https://docs.civicrm.org/user/en/latest/initial-set-up/permissions-and-access-control/" target="_blank" title="Opens documentation in a new window.">Access Control Documentation</a> for more info.</p>
```

Comment:
Needs `{{:: ts() }}`

Reply:
Is there a way to get <strong> or similar into ts?

Reply:
No and IMO that's a good thing. We shouldn't be mixing markup with plain text. I would say just remove the markup as it doesn't really add any value.